ush/templates/FV3LAM_wflow.xml

```diff
@@ -1436,6 +1436,9 @@ MODULES_RUN_TASK_FP script.
     <envar><name>SLASH_ENSMEM_SUBDIR</name><value><cyclestr>{{ slash_ensmem_subdir }}</cyclestr></value></envar>
     <envar><name>SATBIAS_DIR</name><value><cyclestr>&NWGES_BASEDIR;/satbias</cyclestr></value></envar>
     <envar><name>nens</name><value>{{ num_ens_members }}</value></envar>
+    {%- if do_envar_radar_ref and do_envar_radar_ref_once %}
+    <envar><name>OB_TYPE</name><value>conv_dbz</value></envar>
+    {%- endif %}
```

Comment:
Maybe we don't need the "{%- if" block? An extra envar does not hurt anything.

Reply:
@chunhuazhou @hu5970 
After this PR is merged, adjustment of soil T/Q [here](https://github.com/NOAA-GSL/regional_workflow/blob/feature/RRFS_dev1/scripts/exregional_run_postanal.sh#L191) is skipped in the case of SDLVDL. (Is my understanding correct?) I guess this behavior is not as our expectation... Are you thinking about modifying exregional_run_postanal.sh in another PR?

Reply:
@shoyokota Thank you for your comments! You are right that we do need another fix to the run script, in addition to this fix. I have another PR coming in. Thanks!